data/templates/new_game_template

```diff
@@ -0,0 +1,18 @@
+[global]
+interactive_player="1"
+win_condition="scripting/win_conditions/endless_game.lua"
+peaceful="false"
+custom_starting_positions="false"
+map="maps/Lake_of_Tranquility.wmf"
+
+[player_1]
+tribe=
+init="headquarters.lua"
+team="0"
+color="0x161BA7"
```

Comment:
While nice to have i think defining a playercolor should be removed (or commented out). A normal player will not understand how to set this correctly.

Maybe rename it to `playercolor`.

Reply:
Renamed the field, and it's optional now so it can be left out.